README.md

```diff
@@ -85,6 +85,12 @@ The Dart package workflow consists of the following steps:
 
 **Default** `"lib test"`
 
+#### `check_ignore`
+
+**Optional** Allows ignoring lines from [coverage][coverage].
+
+**Default** `true`
```

Comment:
```suggestion
**Default** `false`
```

This is a breaking change as we retag constantly on `v1` so to ensure we dont change the potential output of anyone on the v1 tag we need to make this false